Doc/library/xml.rst

```diff
@@ -75,6 +75,8 @@ decompression bomb         Safe             Safe              Safe             S
 2. :mod:`xml.dom.minidom` doesn't expand external entities and simply returns
    the unexpanded entity verbatim.
 3. :mod:`xmlrpclib` doesn't expand external entities and omits them.
+4. External general entities are no longer processed by default since Python
+   2.7.16, 3.6.7, 3.7.1, and 3.8.0.
```

Comment:
You might only list 3.8.0, but complete this table once we fixed other branches.

I'm not sure about backporting this backward incompatible change to other branches, but if we do it, we should also fix 3.4 and 3.5, no?